doc/usecases/transforming_coordinate_systems.adoc

```diff
@@ -0,0 +1,31 @@
+= Transforming coordinate systems
+
+Transforming coordinates for a specific vehicle or sensor is a common task when running simulations.
+OSI uses DIN ISO 8855:2013-11 for coordinate systems and transformations between coordinate systems.
+
+//TODO: Should we add one or more sentences about the mathematical operations involved?
+
+**Corresponding messages**
+
+``GroundTruth::moving_object::base::position``::
+Defines the position of the vehicle’s reference point in world coordinates.
+In Open Simulation Interface, the position of an object is defined by the coordinates of the center of the object's bounding box.
+
+``GroundTruth::moving_object::base::orientation``::
+Defines the orientation of the vehicle’s reference point in world coordinates.
+
+``GroundTruth::moving_object::vehicle_attributes::bbcenter_to_rear``::
+The vector pointing from the vehicle’s reference point to the middle of the rear axle under neutral load conditions in vehicle coordinates.
+
+``SensorData::mounting_position``::
+Defines the sensor’s position and orientation and thereby the origin of the sensor coordinate system.
```

Comment:
```suggestion
Defines the sensor’s position and orientation and thereby the origin of the sensor's coordinate system.
```

Reply:
This was intentional. I was referring to the _sensor coordinate system_, see https://github.com/OpenSimulationInterface/open-simulation-interface/blob/migration/doc/architecture/reference_points_coordinate_systems.adoc